app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,10 @@
+<tbody>
+  <tr>
+    <td><%= food.name %></td>
+    <td><%= food.measurement_unit %></td>
+    <td> $ <%= food.price%></td>
+    <td>
+        <%= button_to "Delete", food, :method => :delete, data: { confirm: "Are you sure?" } %>
+    </td>
+  </tr>
```

Comment:
[OPTIONAL]
- The delete button in the food card does not function as expected. It shows the following error when I try to remove food due to the dependencies between recipe_foods and foods tables:
<img width="856" alt="image" src="https://user-images.githubusercontent.com/77190203/203049484-c5544405-6172-4977-a241-1905bf5752ae.png">
I highly recommend fixing this issue for a better user experience.